### PR TITLE
Create incremental CVs by adding Deb packages

### PR DESCRIPTION
**Pull request by maximiliankolb**
#### What changes are you introducing?

Add Hammer CLI instructions on how to add packages to a CV version.

#### Why are you introducing these changes? (Explanation, links to references, issues, etc.)

Refs https://community.theforeman.org/t/incremental-update-not-working/38811/6?u=maximilian

#### Anything else to add? (Considerations, potential downsides, alternative solutions you have explored, etc.)

-

#### Checklists

* [ ] I am okay with my commits getting squashed when you merge this PR.
* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [x] Foreman 3.13
* [ ] Foreman 3.12/Katello 4.14 (Satellite 6.16)
* [ ] Foreman 3.11/Katello 4.13
* [ ] Foreman 3.10/Katello 4.12
* [ ] Foreman 3.9/Katello 4.11 (Satellite 6.15; orcharhino 6.8/6.9/6.10)
* [ ] Foreman 3.8/Katello 4.10
* [ ] Foreman 3.7/Katello 4.9 (Satellite 6.14)
* [ ] Foreman 3.6/Katello 4.8
* [ ] Foreman 3.5/Katello 4.7 (Satellite 6.13; orcharhino 6.6/6.7)
* We do not accept PRs for Foreman older than 3.5.

### CODE REVIEW

File: guides/common/modules/proc_creating-an-incremental-content-view-version-by-adding-deb-packages.adoc [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+----
+$ hammer content-view info --id _My_Content_View_ID_
+----
+. Create an incremental content view version by adding Deb packages:
++
+[options="nowrap" subs="+quotes"]
+----
+$ hammer content-view version incremental-update \
+--content-view-version-id _My_Content_View_Version_ID_ \
+--deb-ids _My_Deb_Package_ID_ \
```
[the remark this answers is not in the document]
Comment (m-bucher) [reply guides/common/modules/proc_creating-an-incremental-content-view-version-by-adding-deb-packages.adoc:39]: `hammer content-view version incremental-update --debs _My_Debian_Package_Name_`

The parameter is `--debs` :smile:

**Review by m-bucher — COMMENTED** [review]
Looks good to me.

File: guides/common/modules/proc_creating-an-incremental-content-view-version-by-adding-deb-packages.adoc (hunk 1 above)
[the remark this answers is not in the document]
Comment (m-bucher) [reply guides/common/modules/proc_creating-an-incremental-content-view-version-by-adding-deb-packages.adoc:39]: This was buggy in the past, but fixed in https://github.com/Katello/hammer-cli-katello/pull/960

**Review by m-bucher — APPROVED** [review]

File: guides/common/modules/proc_creating-an-incremental-content-view-version-by-adding-deb-packages.adoc [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
++
+[options="nowrap" subs="+quotes"]
+----
+$ hammer content-view version incremental-update \
+--content-view-version-id _My_Content_View_Version_ID_ \
+--debs _My_Deb_Package_Name_ \
+--lifecycle-environment-ids _My_Lifecycle_Environment_ID_
+----
++
+You can add more Deb package names in a comma-separated list.
```
Comment (Lennonka) [inline guides/common/modules/proc_creating-an-incremental-content-view-version-by-adding-deb-packages.adoc:43]:
```suggestion
You can add multiple Deb package names in a comma-separated list.
```

**Review by Lennonka — COMMENTED** [review]
LGTM. One non-blocker nitpick

**Review by Lennonka — APPROVED** [review]

File: guides/common/modules/proc_creating-an-incremental-content-view-version-by-adding-deb-packages.adoc (hunk 2 above)
Comment (maximiliankolb) [reply guides/common/modules/proc_creating-an-incremental-content-view-version-by-adding-deb-packages.adoc:43]: Applied.

**Comment by maximiliankolb** [comment]
Merged to "master" and cherry-picked to "3.13":
   abfb88a255..3f5def3093  3.13 -> 3.13

**Pull request merged**